### DISCUSSION TITLE
Helper Library for KOP/FUP

### DISCUSSION THREAD

**Discussion by jogibear9988** [simatic-ax/.discussions#9] [General]
Hello,

don't know if KOP/FUP could be used in Simatic AX. But for anyone interested, we created a Helper Library to generate the XML Code of KOP/FUP Blocks from C#.
Maybe someone also has use for it: https://github.com/dotnetprojects/TiaCodeGen

**Comment by sjuergen** [comment]
Hi @jogibear9988 thanks for sharing that information. Currently LAD is is not supported by SIMATIC AX.
